Makefile-tests.am

```diff
@@ -70,18 +70,26 @@ check-local:
 	@echo "  *** NOTE ***"
 
 vmbuild:
-	vagrant up # provision if not already done
-	vagrant rsync
-	vagrant ssh -c "cd sync/tests/vmcheck && make install VERSION=$$(git describe)"
-	vagrant ssh -c "sudo systemctl reboot" || :
-
-vmcheck: vmbuild
-	true # start test harness here
+	@if [ -z "$(SKIP_VMBUILD)" ]; then \
+	  vagrant up && \
+	  vagrant rsync && \
+	  vagrant ssh -c "cd sync/vagrant && \
+	                  make install VERSION=$$(git describe)" && \
+	  ( vagrant ssh -c "sudo systemctl reboot" || : ) && \
+	  sleep 2; \
```

Comment:
We spent a *lot* of time debugging this kind of stuff for the Ansible tests...then ultimately indeed ended up with a timeout =/ 
https://github.com/projectatomic/atomic-host-tests/pull/24